modules/kubefed-registering-clusters.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+// * kubefed/kubefed-resource-federation.adoc
+
+[id="kubefed-registering-clusters_{context}"]
+= Registering (joining) clusters
+
+The `join` operation reads information about how to connect the registering cluster and the cluster hosting federation from your local `kubeconfig` file.
+
+You can use the `kubefedctl join` command to register clusters for {KubeFedProductShortName} resource federation.
+
+.Procedure
+
+. Use the `kubefedctl join` command with the required arguments as shown in the example.
++
+----
+$ kubefedctl join <cluster-name> \
+  --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join (required when the context name differs from the cluster name)
+  --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster (required when the current-context isn't set correctly.)
+  --kubefed-namespace=<namespace> \        # namespace where KubeFed is deployed (required when the kubefed-namespace isn't `kube-federation-system`)
+  -v 2
+----
++
+[NOTE]
+====
+`cluster-context` will default to using the registering cluster name if not specified.
+====
+The `kubefedctl join` command creates a KubeFedCluster resource named `<cluster-name>` to represent the joined cluster.
+KubeFedCluster resources configure the API endpoint of a registered cluster, and reference a local secret containing the authentication token used to connect to that cluster.
+
+. Rename the context so that it is in a {product-title} consumable format using the following command
```

Comment:
Suggest than "Rename the context .." step should be in front of step "Use the `kubefedctl join` command ..."